src/App.js

```diff
@@ -36,19 +39,21 @@ const donations = [
   },
 ];
 
-function App() {
-  return (
-    <>
-      <TopBar />
-      <main className="container">
-        <section className="sidebar">{/* Recent Donations */}</section>
-        <section className="">
-          {/* Progress */}
-          {/* Donation Form */}
-        </section>
-      </main>
-    </>
-  );
+export default class App extends React.Component {
+  render() {
+    return (
+      <>
+        <TopBar />
+        <main className="container">
+          <section className="sidebar">
+            <RecentDonations donations={donations}/>
+          </section>
+          <section className="">
+            <Progress max={targetAmount}/>
+            <DonationForm donations={donations.length+1}/>
+          </section>
+        </main>
+      </>
+    );
+  }
```

Comment:
Im curious as to why you changed this to a class component. As we will be focusing on hooks, which only work with functional components, I would stay away from class components for now.